src/workers/followUp.ts

```diff
@@ -22,7 +26,7 @@ const followUp = new DiscordWorker<FollowUpJob>(
       default: { schema, prompt, queryTexts },
     } = await import(resolve(import.meta.dirname, `../prompts/${type}`))
 
-    const markdown = await vectorDB.getRelevantMarkdown(url, queryTexts, 5)
+    const markdown = await vectorDB.getRelevantMarkdown(url, queryTexts, 15)
```

Comment:
We are also sending more pages - so the correct data could be included next time.